provider/marathon/marathon_test.go

```diff
@@ -424,6 +424,16 @@ func TestMarathonTaskFilter(t *testing.T) {
 			expected: false,
 		},
 		{
+			desc: "single service without port",
+			task: createTask(taskPorts(80, 81)),
+			application: createApplication(
+				appPorts(80, 81),
+				labelWithService(types.LabelPort, "80", "web"),
+				labelWithService(types.LabelPort, "illegal", "admin"),
+			),
+			expected: false,
```

Comment:
Why would this combination result in a task being filtered out?  Do we think that if the task has services, EVERY service MUST have a port or portIndex label?